airflow_config/dags/cornflow_api.py

```diff
@@ -1,6 +1,6 @@
 import requests
```

Comment:
This file should be deleted because now it is here: https://github.com/baobabsoluciones/cornflow-client

Reply:
ok, I'll make a new commit with these changes and some of mine.